src-doc/contenuti/requisiti.tex

```diff
@@ -0,0 +1,160 @@
+\chapter{Requsiti}
+
+\section{Introduzione}
+Di seguito sono riportati i requisiti che il gruppo \textit{SWEasabi} ha classificato secondo quanto definito nelle \textit{Norme di Progetto}.
+
+\section{Requisiti funzionali}
+
+\begin{center}
+    \begin{xltabular}{\linewidth}{|l|l|X|l|}
+        \hline
+        %Colora la riga di un grigio chiaro più chiaro
+        \rowcolor[gray]{0.9}
+        \textbf{Codice} & \textbf{Classificazione} & \textbf{Descrizione} & \textbf{Fonti} \\
+        \hline
+        RF\_01 & Obbligatorio & L'utente deve visualizzare un messaggio d'errore esplicativo in caso di errore di autenticazione & UC01 \\
+
+        \hline
+        RF\_02 & Obbligatorio &Ogni cambiamento di stato di una lampadina deve apparire automaticamente nell'interfaccia utente & VIN\_20230111\\
```

Comment:
obbligatorio? non desiderabile?

Reply:
Questo l'hai fatto tu, se ben ricordo, però concordo che potremmo cambiarlo in desiderabile